rules/CodingStyle/Application/UseImportsAdder.php

```diff
@@ -74,42 +77,48 @@ public function addImportsToStmts(
                 break;
             }
 
-            $nodesToAdd = array_merge([new Nop()], $newUses);
+            $position = $key + 1;
 
-            $this->mirrorUseComments($stmts, $newUses, $key + 1);
+            $useComments = $this->getAndRemoveExistingUseComments($stmts, $position);
 
             // remove space before next use tweak
-            if (isset($stmts[$key + 1]) && ($stmts[$key + 1] instanceof Use_ || $stmts[$key + 1] instanceof GroupUse)) {
-                $stmts[$key + 1]->setAttribute(AttributeKey::ORIGINAL_NODE, null);
+            if (isset($stmts[$position]) && ($stmts[$position] instanceof Use_ || $stmts[$position] instanceof GroupUse)) {
+                $stmts[$position]->setAttribute(AttributeKey::ORIGINAL_NODE, null);
             }
 
-            array_splice($stmts, $key + 1, 0, $nodesToAdd);
+            array_splice($stmts, $position, 0, [new Nop()]);
```

Comment:
Instead of adding a `Nop` node to the list of nodes to add, we just insert it directly here. In that way we know that `$newUses`  is an array of `Use_` statements